frontend/public/components/overview/index.tsx

```diff
@@ -866,11 +866,15 @@ class OverviewMainContent_ extends React.Component<OverviewMainContentProps, Ove
   }
 
   createOverviewData(): void {
-    const {loaded, updateResources} = this.props;
+    const {loaded, resources, updateResources} = this.props;
 
     if (!loaded) {
       return;
     }
+    // keeps deleted bookmarked projects from attempting to generate data
+    if (Object.entries(resources).length === 0) {
```

Comment:
Can we check `this.props.mock` instead?

Reply:
Just pushed the change. Looks like it works.